### DISCUSSION TITLE
Can`t get ESP32 to flash

### DISCUSSION THREAD

**Discussion by adamcdub** [darthcloud/BlueRetro#890] [Q&A]
Can anyone help me out, I`m on my third esp32 and still can`t flash. I updated the drivers and confirmed with device manager it is com4 
![image](https://github.com/darthcloud/BlueRetro/assets/156246634/bd0d019d-862b-43d0-a130-f7cd44fd4867)

**Comment by adamcdub** [comment]
If anyone can help, I just ordered another esp32 and I’m getting the same problem, no XMC Flash detected.

**Comment by darthcloud** [comment]
It's not an error, it flashed fine.

**Comment by adamcdub** [comment]
Thanks Darkcloud for clearing that up but now I’m stuck on another issue.
I wired the GC using the below diagram 
![IMG_0690](https://github.com/darthcloud/BlueRetro/assets/156246634/6b43c509-637e-4716-962f-95d67c8f8ebb) and when I turn the GC on I can connect to the esp32 via 
Blueretro.io but I can’t get the controllers to connect, I tried using ps4,Wii and Wii U pro controllers.
